platform/solr/platform-solr-server-standalone/src/main/webapp/WEB-INF/web.xml

```diff
@@ -164,25 +164,21 @@
     <welcome-file>index.html</welcome-file>
   </welcome-file-list>
 
-  <!-- added this to the solr web.xml for our sso -->
```

Comment:
❗️ It looks like you removed some of Scott's security changes.  Is there a reason you removed the context parameters?  Does 6.6.0 not support them?